src/components/all-content-modal/AllContentModal.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react'
+import { Box, Typography } from '@mui/material'
+
+import { styles } from '~/components/all-content-modal/AllContentModal.styles'
+
+interface AllContentModalProps {
+  icon?: React.ReactElement
```

Comment:
You can just import ReactElement from react

Reply:
Also, add children: ReactNode to props interface

Reply:
fixed